src/Libraries/Microsoft.AspNetCore.Telemetry.Middleware/Logging/LoggingOptions.cs

```diff
@@ -78,10 +76,10 @@ public class LoggingOptions
     /// The default value is 1 second.
     /// </value>
     /// <remarks>
-    /// The number should be above 1 millisecond and below 1 minute.
+    /// The value should be in the range of 1 millisecond to 1 minute.
```

Comment:
this must be Canadian English, like British English, as Americans say "through" ...

Reply:
Canadian English would be "the number should be above 1 millisecond, eey?"

Reply:
according to this 'through' means inclusive
https://english.stackexchange.com/a/118407

in British, 'to' could be either.